README.md

```diff
@@ -45,6 +45,57 @@ To install run `ansible-galaxy install sansible.kafka` or add this to your
 
 and run `ansible-galaxy install -p ./roles -r roles.yml`
 
+### AWS Setup
```

Comment:
Line https://github.com/sansible/kafka/pull/14/files#diff-04c6e90faac2675aa89e2176d2eec7d8R43 should be updated to `version: v1.2`